R/multinomialtestbayesian.R

```diff
@@ -225,12 +225,12 @@ MultinomialTestBayesian <- function(jaspResults, dataset, options, ...) {
 
   # If no variable is selected, adjust column title
   if(is.null(nhyps)){
-    descriptivesTable$addColumnInfo(name = "expected", title = gettext("Expected"), type = numberType)
+    descriptivesTable$addColumnInfo(name = "expected", title = gettext("Expected"), type = "number")
```

Comment:
A number doesn't display nicely when the value passed is an integer right? You would see unnecessary 1.00 and added decimals whenever people adjust the settings for decimals. 

Would it make sense to do something like this?
```suggestion
    descriptivesTable$addColumnInfo(name = "expected", title = gettext("Expected"), type = if (ready && rlang::is_integerish(values)) "integer" else "number")
```
where `hasData` implies that `values` is actually present (although perhaps that's unnecessary since this is all user input?).

Alternatives to `rlang::is_integerish(values)` would be fine, although I think it's a pretty safe call.

Reply:
Yeah I can do that, good idea... I am thinking whether we could make jaspResults do this itself. E.g. providing a new `type` that would format to `"integer"` or `"number"` at the time of filling the table with data. It seems that this is relatively common issue (e.g., df in ANOVAs and t-tests) and not always you have the data to do this check at the time of constructing the table, right?

Reply:
sure, that shouldn't be too hard. We can create a new type for this, but we could also just do this automagically whenever `type = "number"`. Any preferences?

Reply:
I think I lean towards a new type. I am just assuming there may be instances where you may want to display e.g. `1.00` to show that the number is 1 to two decimal places. But I don't know if it's that relevant.

Reply:
I made an issue for this at: https://github.com/jasp-stats/INTERNAL-jasp/issues/1818

Should I still follow up your suggestion here regardless of whether we will make the new column type?